server_smtp.py

```diff
@@ -20,6 +19,28 @@
 # deverá abortar.
 # !SECTION
 
+
+if len(sys.argv) < 2:
+    sys.exit("ERROR: O nome do arquivo não foi passado como argumento!") 
+
+name_arq =  sys.argv[1]
+   
+try:
+    arq = open(name_arq, 'r')
+    users = arq.readlines()
+    users_list = []
+    
+    for user in users:
+        user = user.rstrip('\n')
+        users_list.append(user)
+        users_arq = open(user+".txt", 'w')
+            
+    arq.close()
+    users_arq.close()  
```

Comment:
vários arquivos estarão sendo abertos nesse ponto e apenas o ultimo será fechado, não?
```suggestion
        users_arq = open(user+".txt", 'w')
        users_arq.close()
            
    arq.close()
```